### DISCUSSION TITLE
Update v1.1 🎉

### DISCUSSION THREAD

**Discussion by SwiftyProgrammer690** [SwiftyProgrammer690/fireworks#3] [Announcements]
Update v1.1 has come out with two new features to use!

The new features are:

`get_user(user)` - Get a user profile 
`get_org(org)` - Get a organizations profile

<hr /><em>This discussion was created from the release <a href='https://github.com/SwiftyProgrammer690/fireworks/releases/tag/v1.1'>Update v1.1 🎉</a>.</em>

**Comment by anonymouse-cmd** [comment]
Nice!
